airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -104,13 +105,78 @@ def setup_mysql_db(self):
         ]
         print("Executing: ", " ".join(commands))
         subprocess.call(commands)
-        time.sleep(5)
+        time.sleep(15)
```

Comment:
Good idea, let's add some info on the console when the test does not start right away:

```suggestion
                print(f"....Waiting for Postgres DB to start...15 sec")
                time.sleep(15)
```

Reply:
Sure) Also, added for My SQL)